specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2022-04-01-preview/common.json

```diff
@@ -0,0 +1,91 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-04-01-preview",
```

Comment:
This should be a stable version, since none of the changes in this PR require preview release.
Please fork from latest stable version and create a new PR.